folly/portability/OpenSSL.h

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <openssl/base.h>
```

Comment:
Done. Follow @anirudhvr's comment to remove the inclusion. Sorry that I forget to verify the build (outside our custom build environment) before uploading. :(